src/frontend/packages/lib_components/src/types/apps/classroom/models.ts

```diff
@@ -127,8 +133,14 @@ export interface ClassroomDocument extends Resource {
   url: string;
 }
 
+export type ClassroomRecordingVod = Pick<
+  Video,
+  'id' | 'title' | 'upload_state'
+>;
+
 export interface ClassroomRecording extends Resource {
   classroom: string;
   video_file_url: string;
   started_at: string;
+  vod: Nullable<ClassroomRecordingVod>;
```

Comment:
FMI, why null instead of undefined?

Reply:
Because this key is always sent by the backend, but could be set to None.